### PR TITLE
Respawn swaps on startup of cnd

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: cnd/src/db.rs [hunk 1]
```diff
@@ -51,12 +51,18 @@ pub struct ForSwap<T> {
     pub data: T,
 }
 
-/// Load data from the database.
+/// Load data from the database for a particular swap id.
```
Comment (tcharding) [inline cnd/src/db.rs:54]: Why did you remove this part of the comment? It seems even more relevant with the addition of `LoadAll`.

File: cnd/src/db.rs [hunk 2]
```diff
@@ -51,12 +51,18 @@ pub struct ForSwap<T> {
...
 #[async_trait]
 pub trait Load<T>: Send + Sync + 'static {
     async fn load(&self, swap_id: LocalSwapId) -> anyhow::Result<T>;
 }
 
+/// Load data all data of type T from the database.
```
Comment (tcharding) [inline cnd/src/db.rs:60]:
```suggestion
/// Load all data of type T from the database.
```

File: cnd/src/respawn.rs [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            (
+                Swap {
+                    id,
+                    role,
+                    alpha: Protocol::Herc20,
+                    beta: Protocol::Halight,
+                },
+                Some(lnd_params),
+            ) => {
+                let swap: Swap<herc20::Params, halight::Params> = match facade.load(id).await {
```
Comment (tcharding) [inline cnd/src/respawn.rs:46]: I thought we prefer turbofish over explicit type?

File: cnd/src/db.rs (hunk 1 above)
Comment (thomaseizinger) [reply cnd/src/db.rs:54]: Lol, I added this part of the comment :)

Comment (tcharding) [reply cnd/src/db.rs:54]: ROFL, that's funny.

File: cnd/src/respawn.rs (hunk 3 above)
Comment (thomaseizinger) [reply cnd/src/respawn.rs:40]: I prefer it over typing `Load::load(facade, ...`.
